config.json

```diff
@@ -193,12 +197,12 @@
       "slug": "atbash-cipher",
       "uuid": "82169e85-3938-48da-ada9-7394adadc57a",
       "core": false,
-      "unlocked_by": "rna-transcription",
+      "unlocked_by": "hamming",
       "difficulty": 4,
```

Comment:
I made it so that several exercises are unlocked by earlier ones. I think that'll be more interesting for students and mentors alike

Reply:
This example I choose to comment on, of course, is an exception haha...